src/sentry/static/sentry/app/components/helpSearch.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t, tn} from 'app/locale';
+import Search from 'app/components/search';
+import HelpSource from 'app/components/search/sources/helpSource';
+import SearchResult from 'app/components/search/searchResult';
+import SearchResultWrapper from 'app/components/search/searchResultWrapper';
+import space from 'app/styles/space';
+import {IconWindow} from 'app/icons';
+
+type HelpResult = Parameters<
+  React.ComponentProps<typeof HelpSource>['children']
+>[0]['results'][0];
+
+type ResultItemProps = HelpResult & {
+  highlighted: boolean;
+  itemProps: any;
+};
+
+const renderResult = ({item, matches, itemProps, highlighted}: ResultItemProps) => {
+  const sectionHeading =
+    item.sectionHeading !== undefined ? (
+      <SectionHeading>
+        <IconWindow />
+        {t('From %s', item.sectionHeading)}
+        <Count>{tn('%s result', '%s results', item.sectionCount ?? 0)}</Count>
+      </SectionHeading>
+    ) : null;
+
+  if (item.empty) {
+    return (
+      <React.Fragment>
+        {sectionHeading}
+        <Empty>{t('No results from %s', item.sectionHeading)}</Empty>
+      </React.Fragment>
+    );
+  }
+
+  return (
+    <React.Fragment>
+      {sectionHeading}
+      <SearchResultWrapper {...itemProps} highlighted={highlighted}>
+        <SearchResult highlighted={highlighted} item={item} matches={matches} />
+      </SearchResultWrapper>
+    </React.Fragment>
+  );
+};
+
+// TODO(ts): Type based on Search props once that has types
+const HelpSearch = props => (
+  <Search
+    {...props}
+    sources={[HelpSource]}
+    minSearch={3}
```

Comment:
Please why 3?

Reply:
You have to type at least 3 characters to start getting results.

This prop could maybe use a different name.